code/datums/gamemode/factions/bloodcult/bloodcult_items.dm

```diff
@@ -1922,19 +1922,25 @@ var/list/arcane_tomes = list()
 	desc = "A candle made out of blood moth wax, burns much longer than regular candles. Used for moody lighting and occult rituals."
 	icon = 'icons/obj/candle.dmi'
 	icon_state = "bloodcandle"
+	item_state = "bloodcandle"
+	color = null
 
-	wax = 1200 // 20 minutes
+	wax = 3600 // 60 minutes
```

Comment:
Is the intent here that 1 wax unit = 1 second of burning? If so this should be
`wax = 3600 SECONDS`

Reply:
"SECONDS" is just a define that multiplies your value by 10 and is generally used when the value is passed as arg for a spawn() or sleep().

In this instance, wax gets decremented on process, so the value is already in actual seconds and doesn't need the define added on top. Writing 3600 SECONDS would result in the candle actually burning for 6 hours.

Reply:
Ah; I forgot process() ticks each second, thanks.